### PR TITLE
conf_file..unix_sock: Change test command to net-define

### PR DESCRIPTION
**Pull request by Hao-Liu**
To test whether readonly socket is working, this commit change the virsh
test command from `list` to `net-define`.

Signed-off-by: Hao Liu <hliu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py [hunk 1]
```diff
@@ -75,11 +74,15 @@ def check_unix_sock(group, perms, path):
 
         # Check virsh connection.
         uri = 'qemu+unix:///system?socket=%s' % path
-        result = virsh.dom_list('--all', uri=uri)
+        result = virsh.net_autostart('default', uri=uri)
```
Comment (will-Do) [inline libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py]: Then we need make sure 'default' does exist on the host

Comment (yumingfei) [inline libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py]: Yes, you have to deal with it, even it runs well currently.
And, why should you change it to net-autostart?

Comment (Hao-Liu) [inline libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py]: Well, I just want to change the virsh command to another one which need authorization. Thus I can test the difference between ro and rw sockets. But it seems very difficult to find one that is both easy the implement and reliable. Do you have any suggestions?

Comment (yumingfei) [inline libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py]: Maybe you can try to define your own network in this module.
It's easy to be implemented by providing a fixed network xml.

Comment (Hao-Liu) [inline libvirt/tests/src/conf_file/libvirtd_conf/unix_sock.py]: Sure, I will change to net-define, thanks for the advice.

**Comment by Hao-Liu** [comment]
Changed test command from net-autostart and net-define to prevent potential issue. Thanks @will-Do, @yumingfei.

**Comment by yumingfei** [comment]
Good, thanks. Applied.

**Pull request merged**